runtime/main/main.ll

```diff
@@ -29,7 +29,7 @@ entry:
   %output_str = load i8*, i8** %output_ptr
   store i8* %output_str, i8** @output_file
 
-  call void @__gmp_set_memory_functions(i8* (i64) * @koreAllocOld, i8* (i8*, i64, i64) * @koreReallocOld, void (i8*, i64) * @koreFree)
+  call void @__gmp_set_memory_functions(i8* (i64) * @koreAllocNoGC, i8* (i8*, i64, i64) * @koreReallocNoGC, void (i8*, i64) * @koreFree)
```

Comment:
Can you please refactor this out into a separate method? Right now it is duplicated with the main method of the blockchain vm server and I'd like to centralize the code so that future changes will not require updating both repos.

Reply:
done